pgtoolkit/hba.py

```diff
@@ -317,18 +319,14 @@ class HBA:
     .. automethod:: merge
     """
 
-    lines: list[HBAComment | HBARecord]
-    path: str | Path | None
+    entries: Iterable[HBAComment | HBARecord] | None = None
+    lines: list[HBAComment | HBARecord] = field(default_factory=list, init=False)
+    path: str | Path | None = field(init=False)
```

Comment:
If the `path` attribute is not initialized here or in `__post_init__()`, what is its value? If this is not caught in tests, that's because we're missing some...